AzureAD/microsoft-identity-web
# Jmprieur/adding blazor templates

Adding the Blazor server template
- tested with --auth SingleOrg


To test:
- build the templates (under ProjectTemplates, dotnet pack)
- cd bin\debug
- dotnet new -i Microsoft.Identity.Web.ProjectTemplates.0.2.1-preview.nupkg
- mkdir testblazorserver
- cd testblazorserver
- dotnet new blazorserver2 --auth SingleOrg
- dotnet build

update the app coordinates
run. Should sign-in, sign-out

Need to test with B2C, --calls-graph  and --called-api-uri/--called-api-scopes
